src/client.js

```diff
@@ -19,18 +20,56 @@ class SkynetClient {
    * @param {Object} [customOptions.params={}] - Query parameters to include in the URl.
    */
   constructor(portalUrl, customOptions = {}) {
+    // Check if portal URL provided twice.
+
     if (portalUrl && customOptions.portalUrl) {
       throw new Error(
         "Both 'portalUrl' parameter provided and 'customOptions.portalUrl' provided. Please pass only one in order to avoid conflicts."
       );
     }
 
+    // Add portal URL to options if given.
+
     this.customOptions = { ...customOptions };
     // If portal was not given, the default portal URL will be used.
     if (portalUrl) {
       // Set the portalUrl if given.
       this.customOptions.portalUrl = portalUrl;
     }
+
+    // Re-export selected client methods from skynet-js.
+
+    let browserClient = new BrowserSkynetClient(portalUrl);
+    this.browserClient = browserClient;
+
+    // Download
+    this.getSkylinkUrl = browserClient.getSkylinkUrl.bind(browserClient);
+
+    // File API
+    this.file = {
+      getJSON: browserClient.file.getJSON.bind(browserClient),
+      getEntryData: browserClient.file.getEntryData.bind(browserClient),
+      getEntryLink: browserClient.file.getEntryLink.bind(browserClient),
+      getJSONEncrypted: browserClient.file.getJSONEncrypted.bind(browserClient),
+    };
+
+    // SkyDB
+    this.db = {
+      deleteJSON: browserClient.db.deleteJSON.bind(browserClient),
+      getJSON: browserClient.db.getJSON.bind(browserClient),
+      setJSON: browserClient.db.setJSON.bind(browserClient),
```

Comment:
I don't think this would work since we're uploading a file, right?

Reply:
Perhaps we can just throw an error here?

Reply:
`setJSON` takes a JSON object, not a file. 🙂